exercises/build.sbt

```diff
@@ -4,11 +4,12 @@ version := "0.0.1"
 
 scalaVersion := "2.11.6"
 
-libraryDependencies += "org.specs2" %% "specs2-core" % "3.4" % "test"

```

Comment:
This is, strictly speaking, not necessary. Adding a single dependency with `+=` is the same as adding a sequence containing a single dependency with `++=`.